lib/modules/userHighlight.js

```diff
@@ -186,12 +186,12 @@ modules['userHighlight'] = {
 					hoverColor: this.options.firstCommentColorHover.value
 				}
 			};
-			if (this.options.highlightOP.value) {
-				this.highlight('submitter');
-			}
 			if (this.options.highlightFriend.value) {
```

Comment:
Shouldn't we be flipping this, doing mod, then submitter, then friend? Seems like the latest rule should trump all the rest.